package.json

```diff
@@ -74,6 +74,7 @@
     "react-redux": "7.2.4",
     "react-router-dom": "5.3.3",
     "react-select": "2.4.4",
+    "react-switch-selector": "1.2.1",
```

Comment:
Can we get away without introducing a dependency for this (e.g. with a small component and a transition?). The library seems to not being updated frequently. Also, a quick question: if introducing the library is needed, why have an outdated version (current is 2.2.1)? I see in https://github.com/GR34SE/react-switch-selector/blob/master/package.json that react 16+ is in the peer dependencies, so there shouldn't be an issue related to this.

Reply:
This was brought up to the web platform team and mentioned it was OK as it is a small one, see https://community.mattermost.com/private-core/pl/15ciepgagfn93bi1b1otoory9e